apps/chat-e2e/.env.development

```diff
@@ -2,6 +2,7 @@
 # E2E_HOST=""
 # E2E_USERNAME="" # used for Auth0 E2E login
 # E2E_PASSWORD=""
-# TRACES_URL=""
+# OTEL_EXPORTER_OTLP_ENDPOINT
```

Comment:
Should we add it to `environment.d.ts`?

Reply:
Variable is removed since it not used in the e2e tests (@irinakartun ).
I don't think we need to add Opentelemetry variables to an `environment.d.ts`.
Code changed to support standard Opentelemetry variables.